base_layer/core/src/mempool/service/service.rs

```diff
@@ -238,33 +238,33 @@ impl MempoolService {
         });
     }
 
-    fn spawn_handle_incoming_request(&self, domain_msg: DomainMessage<mempool_proto::MempoolServiceRequest>) {
+    async fn spawn_handle_incoming_request(&self, domain_msg: DomainMessage<mempool_proto::MempoolServiceRequest>) {
         let inbound_handlers = self.inbound_handlers.clone();
         let outbound_message_service = self.outbound_message_service.clone();
-        task::spawn(async move {
-            let result = handle_incoming_request(inbound_handlers, outbound_message_service, domain_msg).await;
+        // task::spawn(async move {
```

Comment:
remove?

Reply:
Thanks -  I think I'll bring them back in and leave it for another refactor to evaluate where we can cut down on these spawns